shared/src/store/action-reducers/simulation.ts

```diff
@@ -1285,12 +1702,34 @@ export namespace SimulationActionReducers {
                 draftState,
                 { simulatedRegionId, behaviorId, managedSimulatedRegionId }
             ) {
+                const simulatedRegion = getElement(
+                    draftState,
+                    'simulatedRegion',
+                    simulatedRegionId
+                );
                 const behaviorState = getBehaviorById(
                     draftState,
                     simulatedRegionId,
                     behaviorId,
                     'managePatientTransportToHospitalBehavior'
                 );
+                const managedSimulatedRegion = getElement(
+                    draftState,
+                    'simulatedRegion',
+                    managedSimulatedRegionId
+                );
+                logBehavior(
+                    draftState,
+                    [
+                        createSimulatedRegionTag(
+                            draftState,
+                            managedSimulatedRegionId
+                        ),
+                    ],
```

Comment:
This tag is duplicated, if the simulated region is managing itself

Reply:
d